pages/about.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Us - Kramerica Industries - Microverse Capstone Project</title>
+    <link type="text/css" rel="stylesheet" href="../styles/styles.css" />
+  </head>
+  <body>
+    <header>
+      <div id="nav-icon">
```

Comment:
- [ ] Please take a look at your hamburger menu on your about page, as it is not working as expected right now. Remember that once you click the hamburger menu, you should be able to see a menu with the links to navigate through the page.